docker/cache/resources/pre-condition-files/008_dataset_def.sparql

```diff
@@ -1,9 +1,57 @@
+PREFIX : <http://linked.data.gov.au/def/loci#> 
 PREFIX dcat: <http://www.w3.org/ns/dcat#>
 
 INSERT DATA {
-    <http://linked.data.gov.au/dataset/gnaf> a dcat:Dataset .
-    <http://linked.data.gov.au/dataset/gnaf-2016-05> a dcat:Dataset .
-    <http://linked.data.gov.au/dataset/geofabric> a dcat:Dataset .
-    <http://linked.data.gov.au/dataset/asgs2016> a dcat:Dataset .
-    <http://linked.data.gov.au/dataset/asgs2011> a dcat:Dataset .
+    GRAPH <http://linked.data.gov.au/dataset/explicit> {
```

Comment:
Presumably this is to distinguish dataset definitions here from dataset definitions made through linksets? I'm not sure.

Reply:
This is really about bringing in the stuff that was in the loci-ont into a new place. I have not changed the content. It should be as broken as before, but more visible now.

Reply:
We really need to have a good discussion about what a Dataset it etc. But that might ball out of the spatial enablement guidelines?